### PR TITLE
Receive events in a unit of 100

### PR DESCRIPTION
**Pull request by syncpark**
Apply to `Tis` branch.

Receive events in a unit of 100.

Discussion https://github.com/aicers/giganto/discussions/671
Pending aicers/piglet#767
Compatible REproduce - https://github.com/aicers/reproduce/tree/tis
Compatible Piglet - https://github.com/aicers/piglet/tree/tis

### CODE REVIEW

**Comment by sehkone** [comment]
@syncpark To-do list에는 PR을 올리지 말고, 이슈를 올려 주세요.

File: CHANGELOG.md [hunk 1]
```diff
@@ -464,6 +470,7 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 - Initial release.
 
+[Unreleased]: https://github.com/aicers/giganto/compare/0.19.0+tis.0.1.0...main
```
Comment (sophie-cluml) [inline CHANGELOG.md:473]: `...main` -> `...tis` 로 변경 되어야 할 것 같아요.

Comment (syncpark) [reply CHANGELOG.md]: Done

**Review by sophie-cluml — APPROVED** [review]
LGTM

**Pull request merged**